soc/riscv32/litex-vexriscv/soc.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2018 - 2019 Antmicro <www.antmicro.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __RISCV32_LITEX_VEXRISCV_SOC_H_
+#define __RISCV32_LITEX_VEXRISCV_SOC_H_
+
+#include "../riscv-privilege/common/soc_common.h"
+#include <generated_dts_board.h>
+
+/* MIE address */
+#define SOC_MIE_IEN                  0x880
```

Comment:
Is this define need anywhere but the IRQ code?  If not, just move it there.

Reply:
I got rid of this define at all.